components/Booking.php

```diff
@@ -279,7 +339,7 @@ public function processPickerForm()
         if (!$this->validatePasses($data, $this->createRules('picker')))
             return;
 
-        $this->pickerStep = array_get($data, 'sdateTime') ? 'info' : 'timeslot';
+        $this->pickerStep = 'info';
```

Comment:
Can we bring back the timeslot partial? Not sure why you removed it but OpenTable and resdiary have something similar.

Reply:
The reason for leaving it out was that all the timeslots are now on the calendar page - so you'd be asking someone to choose it to then choose it again. I haven't looked at the changes you've made so that may no longer be the case...

Reply:
Yea I get you but the idea is to give the customer more options in case they want a different time or maybe the time they selected is fully booked ex. customer selects 6:30 which is fully booked but 7:00 is available.

Reply:
Np. Will work on bringing it back next week.

Reply:
Thanks!